src/GenAPI/Microsoft.DotNet.GenAPI.Task/GenAPITask.cs

```diff
@@ -44,7 +44,7 @@ public class GenAPITask : TaskBase
         public string[]? ExcludeAttributesFiles { get; set; }
 
         /// <summary>
-        /// Include all API's not just public APIs. The default is public only.
+        /// IncludeInclude internal API's. Default is false.
```

Comment:
Typo: double "Include"

Reply:
Updated. Thanks. will update in https://github.com/dotnet/sdk/pull/29471